Google.PowerShell/ReleaseFiles/PatchFromGcs.ps1

```diff
@@ -19,25 +21,30 @@ if (-not $installPath) {
         return
     }
 }
-$installPath = $installPath -replace '"' # Registry values had quotes for some reason
-$googlePowerShellPath = Join-Path $installPath "google-cloud-sdk\platform\GoogleCloudPowerShell"
+$installPath = $installPath -replace '"' # Registry values had quotes. This removes them.
+Push-Location (Join-Path $installPath "google-cloud-sdk\platform")
+$googlePowerShellPath = Resolve-Path "GoogleCloudPowerShell"
 
 if (-not (Test-Path $googlePowerShellPath)) {
-    Write-Error "Can not find Google PowerShell. $googlePowerShellPath does not exist."
+    Write-Error "Can not find Google PowerShell. '$googlePowerShellPath' does not exist."
     return
 }
 
-$pathToOldCmdlets = "$googlePowerShellPath\..\OldPowerShell"
+$pathToOldCmdlets = "GoogleCloudPowerShell-unpatched-backup"
 if (Test-Path $pathToOldCmdlets) {
     Remove-Item $pathToOldCmdlets -Recurse
 }
 Move-Item $googlePowerShellPath $pathToOldCmdlets
 Import-Module "$pathToOldCmdlets/GoogleCloudPowerShell.psd1"
 $bucket = Get-GcsBucket g-cloudsharp-unsignedbinaries
-$zipObject = Find-GcsObject $bucket -Prefix powershell | Sort Name -Descending | Select -First 1
-$tempFile = New-TemporaryFile
-Read-GcsObject $bucket $zipObject.Name -OutFile $tempFile -Force
 
+# Find objects in the powershell directory, and select one most recently created.
```

Comment:
We probably want to be more sophisticated. For now this is OK. But in the future we probably want a prefix like "powershell-nightly-builds/" or similar.